live-examples/css-examples/lists/list-style-type.html

```diff
@@ -42,14 +42,6 @@
 
         <hr>
 
-        <div class="note">
-            <pre><code>@counter-style space-counter {
-    symbols: "\1F680"
-             "\1F6F8"
-             "\1F6F0"
-             "\1F52D";
-    suffix: " "
-}</code></pre>
-        </div>
+        <div class="note"><p><code>space-counter</code> is defined with <a href="https://developer.mozilla.org/docs/Web/CSS/%40counter-style"><code>@counter-style</code></a></p>
```

Comment:
Oh wait, seeing that this will be in an iframe, the anchor will need `target="_parent"`, else this will load inside the iframe. Also, the URL need to be preceded with `//` else it will "inherit" the domain of the iframe.

Lastly, it seems like MDN is not playing nice with the encoded `@` symbol so, perhaps make the URL `//developer.mozilla.org/docs/Web/CSS/@counter-style`